RoadMap_Movie/RoadMap_Movie/Cells/MovieTableViewCell.swift

```diff
@@ -66,33 +68,43 @@ final class MovieTableViewCell: UITableViewCell {
 
     // MARK: - Public methods
 
-    func setupMovieTitle(_ movie: Movies) {
+    func setupMovieTitle(_ movie: Movie) {
         movieTitleLabel.text = movie.title
     }
 
-    func setupOverview(_ movie: Movies) {
+    func setupOverview(_ movie: Movie) {
         overviewLabel.text = movie.overview
     }
 
-    func setupImage(_ movie: Movies) {
-        guard let imageURL = URL(string: "\(UrlRequest.basePosterURL)\(movie.poster)") else { return }
-        posterImageView.load(url: imageURL)
+    func setupImage(_ movie: Movie) {
+        let imageURL = "\(URLParts.basePosterURL)\(movie.posterPath)"
+        networkService.fetchImage(imageURLPath: imageURL) { [weak self] result in
+            guard let self else { return }
+            switch result {
+            case let .success(data):
+                DispatchQueue.main.async {
+                    self.posterImageView.image = UIImage(data: data)
+                }
+            case let .failure(error):
+                print(error.localizedDescription)
```

Comment:
```suggestion
                print(error.localizedDescription)
```
надо что-то придумать